project/ProjectPlugin.scala

```diff
@@ -358,13 +403,12 @@ object ProjectPlugin extends AutoPlugin {
         organizationHomePage = url("http://47deg.com"),
         organizationEmail = "hello@47deg.com"
       ),
-      scalaVersion := V.scala,
-      crossScalaVersions := Seq(V.scala),
-      scalacOptions ++= customScalacOptions,
+      scalaVersion := V.scala213,
+      crossScalaVersions := Seq(V.scala212, V.scala213),
+      scalacOptions ~= (_ filterNot Set("-Xfuture", "-Xfatal-warnings").contains),
```

Comment:
Why are we disabling `fatal-warnings`?

Reply:
Also you can write this more simply as `scalacOptions --= Seq("-Xfuture", "-Xfatal-warnings")`

Reply:
> Why are we disabling fatal-warnings?

Current code won't compile as it is. If we want to enable it, I propose to do it in a separate PR.

> Also you can write this more simply as scalacOptions --= Seq("-Xfuture", "-Xfatal-warnings")

👍 https://github.com/higherkindness/mu-scala/pull/807/commits/263c3a5fc8426d3ef42772e41a9a9e9c0bb8e5b6